tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -2489,6 +2489,20 @@ public function foo($foo): static {}
 }',
         ];
 
+        yield 'static return for arrow function' => [
+            '<?php
+    /**
+     */
+    $a = fn ($foo): int => 1;
+',
+            '<?php
+    /**
+     * @return intgit st
+     */
+    $a = fn ($foo): int => 1;
```

Comment:
see https://github.com/phpstan/phpstan/issues/7345#issuecomment-1848966265, such syntax is not supported by phpstan and probably never will

Reply:
🤷🏻

Reply:
also please add tests for these: https://3v4l.org/KFV46

Reply:
I won;'t be able to pick it up in upcoming days. If you are happy to take over the PR @mvorisek , please do not hesitate!

(if not really your pair of shoes, also OK , it will just calmly await )